### PR TITLE
Remove outdated FIXMEs and add recommended properties for a test on FIPS

### PR DESCRIPTION
**Pull request by fedinskiy**
### Summary

See the title

Please select the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [ ] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [x] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by michalvavrik** [comment]
looks good, but OTel also is subclass for OCP test so we need to run OCP.

**Comment by michalvavrik** [comment]
run tests

**Comment by michalvavrik** [comment]
run tests

**Comment by fedinskiy** [comment]
@michalvavrik I did the major changes in Inifinispan test (fixed it instead of disabling), so please take another look

**Comment by michalvavrik** [comment]
> @michalvavrik I did the major changes in Inifinispan test (fixed it instead of disabling), so please take another look

I read QUARKUS-4612, changes are good. I trust you tested it in FIPS-enabled env with Mandrel, so let's get this in.

**Comment by michalvavrik** [comment]
run tests

File: cache/infinispan/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
 quarkus.infinispan-client.hosts=localhost:11222
 quarkus.cache.infinispan.expiring-cache.lifespan=3s
 quarkus.cache.infinispan.expiring-cache.max-idle=5s
+quarkus.native.auto-service-loader-registration=true
```
Comment (michalvavrik) [inline cache/infinispan/src/main/resources/application.properties:4]: My understanding is that this is a workaround QUARKUS-4612 so you should leave a comment as we don't want to keep it

Comment (fedinskiy) [reply cache/infinispan/src/main/resources/application.properties]: Exactly! Added the comment.

**Review by michalvavrik — APPROVED** [review]

**Review by michalvavrik — APPROVED** [review]

**Pull request merged**